kubernetes/samples/scripts/create-oracle-db-service/README.md

```diff
@@ -0,0 +1,63 @@
+# Creating an Oracle Database service and RCU schema for a Fusion Middleware domain
+
+The sample scripts in this directory demonstrate how to:
+* Start an Oracle Database (DB) service in a Kubernetes cluster.
+* Stop an Oracle DB service in a Kubernetes cluster.
+
+## Start an Oracle Database service in a Kubernetes cluster
+
+Use this script to create an Oracle Database service in a Kubernate Namespace with the default credentials, in the Oracle Database Slim image.
+
+The script assumes that either the image, `container-registry.oracle.com/database/enterprise:12.2.0.1-slim`, is available in the Docker repository, or an `ImagePullSecret` is created for `container-registry.oracle.com`. To create a secret for accessing `container-registry.oracle.com`, see the script `create-image-pull-secret.sh`.
+
+```
+
+$ ./start-db-service.sh -h    
```

Comment:
./start-db-service.sh -h  command output should have the -n flag info.

    -n Configurable Kubernetes Namespace for Oracle DB Service  (optional)"
         (default: default) "

Reply:
Added the the information about -n flag